springql-core/src/api/spring_config.rs

```diff
@@ -179,6 +180,7 @@ pub struct SpringWebConsoleConfig {
 pub struct SpringSourceReaderConfig {
     pub net_connect_timeout_msec: u32,
     pub net_read_timeout_msec: u32,
+    pub net_address: String,
```

Comment:
You've added a String field. Then `SpringSourceReaderConfig` can no longer implement `Copy`.

Could you fix the compile error and add a changelog line in `### Changed` section?

Reply:
I put the change into the `### Fixed` section, is that ok?